README.md

```diff
@@ -41,6 +41,27 @@ Container images for bomctl can be found on [Docker Hub](https://hub.docker.com/
 docker run bomctl/bomctl:latest --help
 ```
 
+### Install From Source
+
+Installing bomctl requires the following:
+
+- [Go](https://go.dev/dl/)
+- [Git](https://git-scm.com/downloads)
+- [Make](https://www.gnu.org/software/make/manual/make.html)
+
+Clone the bomctl repository
+
+``` shell
+git clone https://github.com/bomctl/bomctl.git
+cd bomctl
+```
+
+Build using the `Makefile`
+
+```shell
+make build
+```
```

Comment:
The way the Makefile is currently set up, this would build all of the binaries. Do you think it would make sense to add instructions for how to build only a single binary pertaining to the user's OS?

e.g. `make build-macos-apple`, `make build-linux-amd`, `make build-windows-amd`, etc?